styles/globals.css

```diff
@@ -46,11 +48,21 @@ footer {
   text-align: center;
   position: relative;
   bottom: 0;
-  padding-top: 20px;
-  margin-bottom: 100px;
+  padding: 20px 0px;
+  background-color: #041484;
+  color: #f4f4f4;
+  font-weight: 100;
+  margin-top: 10px;
+}
+
+@media only screen and (min-width: 900px) {
+  footer {
+    position: absolute;
+  }
 }
 footer a {
   text-decoration: underline;
+  z-index: -1;
```

Comment:
I used media query for footer using position absolute  for desktop or laptop screens.
To lower the priority of github link provided in anchor tag footer i used z-index: -1 so that due to positioning it doesnt get effected.
even if i remove it doesnt change view though and should i remove it.